src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Tables/ChangeFeed.sql

```diff
@@ -9,14 +9,15 @@
 **************************************************************/
 CREATE TABLE dbo.ChangeFeed (
     Sequence                BIGINT IDENTITY(1,1) NOT NULL,
-    Timestamp               DATETIMEOFFSET(7)    NOT NULL DEFAULT SYSDATETIMEOFFSET(), -- Automatically populate
+    Timestamp               DATETIMEOFFSET(7)    NOT NULL DEFAULT SYSDATETIMEOFFSET(),  -- Automatically populate
     Action                  TINYINT              NOT NULL,
     StudyInstanceUid        VARCHAR(64)          NOT NULL,
     SeriesInstanceUid       VARCHAR(64)          NOT NULL,
     SopInstanceUid          VARCHAR(64)          NOT NULL,
     OriginalWatermark       BIGINT               NOT NULL,
     CurrentWatermark        BIGINT               NULL,
-    PartitionKey            INT                  NOT NULL DEFAULT 1    --FK
+    PartitionKey            INT                  NOT NULL DEFAULT 1,                    -- FK
```

Comment:
Should FilePath part of include column for retrieval purpose?

Reply:
I don't think we query a subset of the ChangeFeed table in any query, except perhaps the UPDATEs we perform. So if we were to add INCLUDE columns, then we would probably add those columns instead.